data/remnant/remnant.txt

```diff
@@ -350,6 +350,11 @@ phrase "remnant"
 phrase "remnant uncontacted"
 	word
 		"(They sent you a hail consisting of a sign language you don't understand.)"
+		"(You hear a short song about how scanning them will be considered a hostile action.)"
+		"If you value the continued survival of humanity, do not scan our ships."
+		"If you value your continued survival, do not scan our ships."
+		"We value our privacy, and will respond to scanning with deadly force."
+		"Do not scan our ships."
```

Comment:
This could maybe use a mention that they're using their singing-esque voice.
```suggestion
		`"If you value the continued survival of humanity, do not scan our ships," a strangely melodic voice says.`
		`"If you value your continued survival, do not scan our ships." The voice speaks with a tone reminiscent of singing.`
		`"We value our privacy, and will respond to scanning with deadly force." The voice is strangely melodic.`
		`"Do not scan our ships," someone sings to you.`
```
Or are these text displays? I don't know.

Reply:
I've generally assumed that they are text transmissions, since on a solar-system wide scale, voice really has no benefit over text. But yeah, most hails are very unclear whether it is voice or text or anything.

I'm not really sure if I like having it spelled out that they are singing, or whether it is better being ambiguous.